src/python/grpcio/grpc/__init__.py

```diff
@@ -649,6 +649,10 @@ def unary_unary(
 
     Args:
       method: The name of the RPC method.
+      request_serializer: Optional behaviour for serializing the request
+        message. Request goes underserialized in case None is passed.
```

Comment:
I don't think this line should have changed from what it was in the last revision - the `request_serializer` serializes, so if it is not passed, the request is not _serialized_.